app/components/app_patient_page_component.rb

```diff
@@ -56,19 +56,27 @@ def vaccination_records
   end
 
   def default_vaccinate_form
-    pre_screening =
+    today_pre_screenings =
       patient_session
         .pre_screenings
         .joins(:session_date)
         .merge(SessionDate.today)
         .order(created_at: :desc)
-        .first
+
+    feeling_well =
+      today_pre_screenings
+        .detect { |ps| ps.feeling_well.present? }
+        &.feeling_well
```

Comment:
```suggestion
    feeling_well = today_pre_screenings.any?(&:feeling_well)
```

I'm struggling a bit to understand this logic, but I think this suggestion has the same behaviour.

Reply:
Oh much easier!

Reply:
If nothing is present, then it sets the value for "feeling_well" and "not_pregnant" to false rather than nil.. I believe that is incorrect?

Reply:
Yes thanks, silly me!